pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -171,6 +171,22 @@ class CollectionNamespaceSerializer(serializers.Serializer):
     name = serializers.CharField(source="namespace")
 
 
+class CollectionVersionSignatureSerializer(serializers.ModelSerializer):
+    """
+    A serializer for the signatures on a Collection Version.
+    """
+
+    signature = serializers.SerializerMethodField()
+
+    def get_signature(self, obj) -> str:
+        # If I check for signing service I would incur a N+1 db issue, assuming ascii for now
+        return bytes(obj.data).decode("utf-8")
+
+    class Meta:
+        model = models.CollectionVersionSignature
+        fields = ("signature", "pubkey_fingerprint")
```

Comment:
can we add signing-service.name here? it will be useful to see via API if a signature is one created within the system or synced from a remote.

Reply:
It would be also useful to have the pulp-created time surfaced here for debugging purposes.